Nytimes project completion

mkezfarias opened the pull request:
We are submitting the positioning and floats project from HTML and CSS course from Microverse curriculum for review so we can merge to master.

paulgureghian committed:
Added a .gitignore file

paulgureghian committed:
Added Stickler config files

paulgureghian committed:
Added html & css files

paulgureghian committed:
Edited README.md

paulgureghian committed:
Edited html file

paulgureghian committed:
Update README.md

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited README.md

mkezfarias committed:
html first half done

paulgureghian committed:
Edited html & css files

mkezfarias committed:
html paragraphs finished, started related links

paulgureghian committed:
Edited html file

paulgureghian committed:
Renamed image file

paulgureghian committed:
Corrected image path

mkezfarias committed:
building footer

mkezfarias committed:
footer 2

paulgureghian committed:
Edited html file

mkezfarias committed:
fixed logos

mkezfarias committed:
finished html part of project

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited css file

paulgureghian committed:
Edited html and css files

mkezfarias committed:
lunch break

mkezfarias committed:
advanced nav bar and head css

mkezfarias committed:
css from top to create-account

mkezfarias committed:
css from top to create-account

mkezfarias committed:
css completed before related articles

mkezfarias committed:
css related stories

mkezfarias committed:
end of day commit

paulgureghian committed:
Edited html & css files

mkezfarias committed:
finished css

mkezfarias committed:
fixed html and css errors on validator

mkezfarias committed:
fixed css issues

mkezfarias committed:
Merge branch 'master' into nytimes

mkezfarias committed:
fixed css sticler errors

ghost commented:
[**DeepCode Report (#74ff5f)**](https://www.deepcode.ai/app/gh/paulgureghian/The_New_York_Times/96b1d3abaaad564a64f9081e6804a3583152a2fa/paulgureghian/The_New_York_Times/74ff5f748e30ea4d9b92bc0a9d578b0c809fc2fe/_/%2F/code/)
---
DeepCode analyzed this pull request.
There are no new issues.

Redvanisation reviewed (changes requested):
### Hello @mkezmkez  and @paulgureghian   🌟

**Good work on this project, but we need to fix the issues below before moving on ⬇️** 

1. The `<aside>` elements (editor's picks and signup) are showing on top of the main content, please apply the proper styles to fix that.
<img width="1573" alt="Screen Shot 2019-10-02 at 10 51 49 AM" src="https://user-images.githubusercontent.com/33462180/66068837-2df1e180-e503-11e9-829d-25bd0a5b612a.png">

3. Please Validate and fix the HTML markup [here](https://validator.w3.org/#validate_by_input)

4. Make the README file more descriptive, check [this](https://github.com/othneildrew/Best-README-Template) link for reference

### _After making the required changes please submit a new code review request_

### Good luck guys! 🍀

mkezfarias committed:
fixed positioning issues

paulgureghian committed:
Edited html file

paulgureghian committed:
Edited css file

mkezfarias committed:
fixed positioning issues

mkezfarias committed:
fixes to positioning issues on bigger screens

Merge branch 'nytimes' of github.com:paulgureghian/The_New_York_Times into nytimes

mkezfarias committed:
fixed positioning issues

mkezfarias committed:
fixed positioning issues3

mkezfarias committed:
fixed positioning issues4

mkezfarias committed:
fixing stickler issues

wathigo reviewed (approved):
## Hi Devs :wave:  
### Great work on the project!
Please remove the large space on the bottom of the page
![Screenshot from 2019-10-03 03-40-38](https://user-images.githubusercontent.com/44072711/66108099-d8afe180-e58f-11e9-983c-4075c1be44ae.png)

**You may move on :rocket:** 
_Happy coding_

mkezfarias commented:
updated
